Tally.h

```diff
@@ -0,0 +1,27 @@
+/* 
+ @file      Tally.h
+ @brief     contains Tally class
+ @author    Luke Eure
+ @date      January 8 2016
+*/
+
+#ifndef TALLY_H
+#define TALLY_H
+
+class Tally {
+private:
+
+    /** count stored in the tally */
+    double _tally_count;
```

Comment:
Right now this ``Tally`` class seems rather pointless -- it seems it just does the job of a double but with added headache. So to make it less pointless let's change it to have another attribute -- ``_tally_squared``. Every time you call  ``Tally::add`` add the square of the addition value to ``_tally_squared``. I'll show you why later.